docs/visual-basic/language-reference/error-messages/type-of-optional-value-for-optional-parameter-is-not-cls-compliant.md

```diff
@@ -49,4 +49,4 @@ A procedure is marked as `<CLSCompliant(True)>` but declares an [Optional](../..
 -   If you are interfacing with Automation or COM objects, keep in mind that some types have different data widths than in the [!INCLUDE[dnprdnshort](~/includes/dnprdnshort-md.md)]. For example, `int` is often 16 bits in other environments. If you are accepting a 16-bit integer from such a component, declare it as `Short` instead of `Integer` in your managed [!INCLUDE[vbprvb](~/includes/vbprvb-md.md)] code.  
   
 ## See Also  
- [\<PAVE OVER> Writing CLS-Compliant Code](http://msdn.microsoft.com/en-us/4c705105-69a2-4e5e-b24e-0633bc32c7f3)
+ 
```

Comment:
this see also section is now empty

Reply:
My preference here is to just delete the ##See Also because there doesn't seem to be any current article that talks about this.

Reply:
@mikeblome, "Writing CLS-Compliant Code" is now [Language independence and language-independent components](https://docs.microsoft.com/en-us/dotnet/standard/language-independence).

Reply:
@rpetrusha Thanks for that! I checked and all the topics that had the PAVE link are error topics -- 35 total instances. Looking through them it seems that they all already have the correct "Language Independence.." link up in the text so it's not really needed in the See Also. To avoid complicating this PR, I propose that it be approved with existing deletions for the PAVE links, and then I do a second PR immediately to (1) find all topics that mention CLS and make sure the correct link is provided either in text or See Also and (2) remove empty See Also sections.

Reply:
That sounds good, @mikeblome.